README.md

```diff
@@ -36,36 +36,49 @@ See usage demo notebooks in ./nbs
 ## Command line usage
 
 ```
-usage: python -m stability_sdk.client [-h] [--height HEIGHT] [--width WIDTH]
-	  								  [--cfg_scale CFG_SCALE] [--sampler SAMPLER] [--steps STEPS]
-									  [--seed SEED] [--prefix PREFIX] [--no-store]
-									  [--num_samples NUM_SAMPLES] [--show]
-									  prompt [prompt ...]
+usage: python -m stability_sdk [-h] [--height HEIGHT] [--width WIDTH] [--start_schedule START_SCHEDULE]
+                 [--end_schedule END_SCHEDULE] [--cfg_scale CFG_SCALE] [--sampler SAMPLER]
+                 [--steps STEPS] [--seed SEED] [--prefix PREFIX] [--engine ENGINE]
+                 [--num_samples NUM_SAMPLES] [--artifact_types ARTIFACT_TYPES]
+                 [--no-store] [--show] [--init_image INIT_IMAGE] [--mask_image MASK_IMAGE]
+                 [prompt ...]
 
 positional arguments:
   prompt
 
-optional arguments:
+options:
   -h, --help            show this help message and exit
   --height HEIGHT, -H HEIGHT
                         [512] height of image
   --width WIDTH, -W WIDTH
                         [512] width of image
+  --start_schedule START_SCHEDULE
+                        [0.5] start schedule for init image (must be greater than 0, 1 is full strength
```

Comment:
I'd find the description easier to read if a semi-colon or period is used (in this exact situation, maybe a semi-colon is better here.. since `0. 1` would almost look like `0.1`).
E.g. something like:
[0.5] start schedule for init image (Must be greater than 0; 1 is full strength
                        text prompt, no trace of init image)